server/journeys/screen/screen/controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+import { sortAscending } from '../../../utils/utils'
+
+export class ScreenController extends BaseJourneyController {
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const outcomeTypeItems = this.sortRadios(
+      await this.getReferenceDataOptionsForRadios(
+        req,
+        'outcome-type',
+        res.locals.formResponses?.['outcomeType'] || req.journeyData.saferCustodyScreening?.outcomeType || '',
```

Comment:
That last `|| ''` isn't needed here. It'll coalesce to undefined which won't match anything in the list anyway